core/lyrics/sources.go

```diff
@@ -25,17 +30,22 @@ func fromExternalFile(mf *model.MediaFile, suffix string) (model.LyricList, erro
 	contents, err := os.ReadFile(externalLyric)
 
 	if errors.Is(err, os.ErrNotExist) {
+		log.Trace(ctx, "no lyrics found at path", "path", externalLyric)
 		return nil, nil
 	} else if err != nil {
 		return nil, err
```

Comment:
In this case (ex: lack of permissions), should we return the error, or just log it as log.Error and return nil, nil? As it is right now, if the user configure priority as `.lrc,embedded`, and there is a permission error, if does not try the embedded version.

Reply:
If permission error, I would consider this a real error. This will be logged in `lyrics.go` and then fallthrough instead.